app/themes/starwars/templates/layouts/_base.html

```diff
@@ -1,6 +1,3 @@
 {% extends 'layouts/_base.html' %}
 
-{% block header %}
-  {% include 'partials/topbar.html' %}
-  <!-- Theme selected: Star Wars -->
-{% endblock %}
+
```

Comment:
Sorry to be a pain, can we remove these empty lines.
Also if you add `<!-- Theme selected: Star Wars -->` then the failing test will pass.